src/main/resources/assets/js/analytics-tracking.js

```diff
@@ -0,0 +1,67 @@
+// Works in combination with the following data-attributes
```

Comment:
This file seems to be a copy of the code at https://github.com/alphagov/gaap-analytics/tree/master/src/. Can we add those scripts as a dependency rather than copying them into our project, so that we can take advantages of any updates to it? I'm probably missing something here.

Reply:
Correct, it’s a compiled version of this. We can’t use this out of the box because it’s written in ES6 so we’d need Babel to compile to ES5.

Reply:
As discussed, given that we cannot include this as a dependency can we add a comment to the file explaining that this code comes from https://github.com/alphagov/gaap-analytics/tree/master/src/. We should consider including this dependency properly if/when we come to refactoring the frontend.